pyvrp/solve.py

```diff
@@ -212,8 +221,17 @@ def solve(
             ls.add_perturbation_operator(perturb_op(data))
 
     pm = PenaltyManager.init_from(data, params.penalty)
-    init = ls(Solution(data, []), pm.booster_cost_evaluator())  # type: ignore
 
-    ils_args = (data, pm, rng, ls, init, params.ils)
+    if initial_solution is None:
+        # Using large penalties will help to find a feasible solution.
+        max_penalty = params.penalty.max_penalty
+        cost_eval = CostEvaluator(
+            [max_penalty] * data.num_load_dimensions,
+            max_penalty,
+            max_penalty,
+        )
```

Comment:
Should we move this into a method on the `PenaltyManager`? Perhaps something like a `fixed_cost_evaluator` with a `penalty: int` value argument? We could then do something like
```python
cost_eval = pm.fixed_cost_evaluator(params.penalty.max_penalty)
```
or similar.

Reply:
I think what bothers me about this is that we're now directly constructing a cost evaluator, whereas we already have an object managing all that.

Reply:
Or a `pm.max_cost_evaluator()` method, if we don't want to be able to specify the values and just want an evaluator with max penalties.